docs/samples/python/example_csv.py

```diff
@@ -0,0 +1,117 @@
+import csv
+import pprint
+import collections
+from typing import List, Iterable, Optional, Union, Dict
+
+from presidio_analyzer import AnalyzerEngine, BatchAnalyzerEngine, RecognizerResult, DictAnalyzerResult
+from presidio_anonymizer import AnonymizerEngine
+from presidio_anonymizer.entities import EngineResult
+
+"""
+Example implementing a CSV analyzer
+
+This example shows how to use the Presidio Analyzer and Anonymizer
+to detect and anonymize PII in a CSV file.
+It uses the BatchAnalyzerEngine to analyze the CSV file, and the
+BatchAnonymizerEngine (defined in batch_processing sample) to anonymize
+the requested columns.
+
+Content of csv file:
+id,name,city,comments
+1,John,New York,called him yesterday to confirm he requested to call back in 2 days
+2,Jill,Los Angeles,accepted the offer license number AC432223
+3,Jack,Chicago,need to call him at phone number 212-555-5555
+
+"""
+
+
+class CSVAnalyzer(BatchAnalyzerEngine):
+
+    def analyze_csv(
+        self,
+        csv_full_path: str,
+        language: str,
+        keys_to_skip: Optional[List[str]] = None,
+        **kwargs,
+    ) -> Iterable[DictAnalyzerResult]:
+
+        with open(csv_full_path, 'r') as csv_file:
+            csv_list = list(csv.reader(csv_file))
+            csv_dict = {header: list(map(str, values)) for header, *values in zip(*csv_list)}
+            analyzer_results = self.analyze_dict(csv_dict, language, keys_to_skip)
+            return list(analyzer_results)
+
+
+class BatchAnonymizerEngine(AnonymizerEngine):

```

Comment:
we should add this to the anonymizer package at a certain point (not for this PR)

Reply:
agree. I'll add it in a seperate PR with supporting unit tests

Reply:
thanks :) happy to help too